server/routes/updateChain.ts

```diff
@@ -69,9 +69,16 @@ const updateChain = async (
     custom_domain,
     default_summary_view,
     terms,
-    snapshot,
   } = req.body;
 
+  let snapshot = req.body['snapshot[]'];
```

Comment:
why don't you destructure snapshot with the rest of req.body above?

Reply:
Because it's a string array the field is literally the string 'snapshot[]' instead of snapshot, so the destructuring won't work here. You can see the same pattern on line 120 for 'featured_topics[]', which was the only other string[] column before we made these snapshot changes.